python-lib/language_support.py

```diff
@@ -84,3 +84,47 @@
 
 Models with Creative Commons licenses are not included because this plugin is licensed under Apache-2
 """
+
+SPACY_LANGUAGE_MODELS_LEMMATIZATION = ["en", "es", "nb", "fr", "de", "ru", "pl"]
+
```

Comment:
For the docstring to be recognized as linked to a variable, it must be directly below with no newline

Reply:
I did it here 09f5159